### DISCUSSION TITLE
FAILED TO CONNECT TO YOUR CODESPACE

### DISCUSSION THREAD

**Discussion by loayalsaid1** [community/community#44456] [Codespaces]
### Select Topic Area

Bug

### Body

I AM TRIENG TO CONNECT TO CS50 CODESPCASE BUT IT KEEP RELOADING AND SAYS,"FAILED TO CONNECT TO CONNECT TO YOUR CODESPACE

**Comment by ThanksOihyuo** [comment]
ььььььььььььььььь

**Comment by salvador-barboza** [comment]
Hi @loayalsaid1! Sorry to hear you are running into issues when connecting to your Codespace. If you keep facing this issue, the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).
